functions/default-vpc.js

```diff
@@ -22,9 +22,11 @@ module.exports = function(event, context) {
       ec2.describeRouteTables({ Filters: [{ Name: 'vpc-id', Values: [info.VpcId] }] }).promise()
     ]);
   }).then((results) => {
-    info.AvailabilityZones = results[0].Subnets.map((subnet) => subnet.AvailabilityZone);
-    info.AvailabilityZoneCount = results[0].Subnets.length;
-    info.PublicSubnets = results[0].Subnets.map((subnet) => subnet.SubnetId);
+    var publicSubnets = results[0].Subnets.filter((subnet) => subnet.MapPublicIpOnLaunch);
```

Comment:
nitpick (feel free to ignore 🌸): Want to use `let` instead of `var`?

Reply:
hehe, I actually typed `let` at first, before I noticed that the rest of the file uses `var`! <3

Reply:
my bad, I should have looked at that. 👍 to sticking to existing style.